catalog/app/containers/Admin/Buckets/Buckets.tsx

```diff
@@ -780,7 +787,15 @@ function PrimaryCard({ bucket, className, disabled, onSubmit }: PrimaryCardProps
             <PrimaryForm bucket={bucket} />
           </form>
           <StickyActions parentRef={ref}>
-            <CardActions<PrimaryFormValues> disabled={disabled} form={form} />
+            <CardActions<PrimaryFormValues>
+              action={
+                <StyledLink to={configHref}>
+                  Modify bucket appearance via {configUrl}
```

Comment:
```suggestion
                  Edit Bucket UI config at {configUrl}
```

maybe smth like this?

Reply:
The fact that the user edits config is our implementation. The user's intention would be to change the appearance of the web app, config is a tool, and we hide the config with the new config editor. So, "modify appearance" is a more user-friendly phrase than "edit config", in my opinion.

Reply:
"config" means "configuration", not "configuration file", and in my opinion "Bucket UI configuration" is somewhat more precise than "bucket appearance", bc it's not so much about "appearance" of a "bucket", but more about the features exposed to the user via our UI. also, this is meant for admins, not end-users, so i don't think they will be scared by that.
"Configure Bucket UI" may be even better (not sure if we should display the s3 uri of the config at all tbh).

in the end, it's your call, but to me the current wording feels kinda awkward